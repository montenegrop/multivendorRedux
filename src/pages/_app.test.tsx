import { mount } from "enzyme"

describe("_app", () => {
  let Comp

  beforeEach(() => {
    Comp = require("./_app").default
  })

  it("render without errors", () => {
    const cmp = mount(<Comp Component={() => <div>Page example</div>} pageProps={{}} />)
    expect(cmp).toMatchSnapshot()
  })
})
