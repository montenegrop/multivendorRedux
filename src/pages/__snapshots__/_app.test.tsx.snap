// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_app render without errors 1`] = `
<MyApp
  Component={[Function]}
  pageProps={Object {}}
>
  <Provider
    store={
      Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <Component>
      <div>
        Page example
      </div>
    </Component>
  </Provider>
</MyApp>
`;
